lib/livebook/hubs/team_client.ex

```diff
@@ -2,6 +2,7 @@ defmodule Livebook.Hubs.TeamClient do
   use GenServer
   require Logger
 
+  alias Livebook.Apps.TeamsAppSpec
```

Comment:
Nitpick: we use the alias only once, so I would just use it fully qualified instead :)

Reply:
I did it for readability, because the `for do` block keeps it in the same line, but I'll add the `Livebook.Apps` alias, since we've been using it in this module.